### ISSUE TITLE
Release mypy 0.511

### ISSUE THREAD

**Issue by JukkaL** [python/mypy#3343]
This will only include fixes to crash bugs #3333 and #3332 that are regressions from 0.501. This is going to be an almost silent release. I may add a small update to the blog post.

**Comment by JukkaL** [comment]
I uploaded the 0.511 release. Can somebody test installation and running it on Windows? Once it's confirmed to work on Windows, the release is finished and we can continue to merge PRs normally.

**Comment by ethanhs** [comment]
I can confirm the release works on Windows. 🎉

**Comment by ErikBjare** [comment]
I've been getting weird errors in one of my [AppVeyor builds](https://ci.appveyor.com/project/ErikBjare/activitywatch/build/1.0.142/job/14m2ir2ppm07jc5x#L2541).

Will check and see if using 0.510 instead resolves it. [Build is running here](https://ci.appveyor.com/project/ErikBjare/activitywatch/build/1.0.144/job/5c46bk2929htjn2r).

**Comment by gvanrossum** [comment]
@ErikBjare If that persists please open a separate issue.

**Comment by JukkaL** [comment]
Feature freeze is now over.
